redhat-cop/containers-quickstarts
# Add install of oc, kubectl, helm to python-kopf-s2i

#### What is this PR About?

This adds oc, kubectl, and helm to base python-kopf-s2i image and bump Python version to 3.8.3.

The addition of these command line tools is a convenience for developers using common design patterns such as leveraging OpenShift templates, kustomize, or helm.

This PR will also trigger release of python-kopf-s2i version v0.28.1

#### How do we test this?

Automatic testing should suffice. For more testing, see the example included under build-s2i-python-kopf/example/.

cc: @redhat-cop/day-in-the-life
